src/components/loader-spinner/loader-spinner.style.ts

```diff
@@ -0,0 +1,152 @@
+import styled, { css } from "styled-components";
+import { margin } from "styled-system";
+import { LoaderSpinnerProps } from "./loader-spinner.component";
+import {
+  LoaderSpinnerSizes,
+  LOADER_SPINNER_SIZE_PARAMS,
+} from "./loader-spinner.config";
+import Typography from "../typography";
+
+const calculateColors = (isWheel: boolean, variant?: string) => {
+  switch (variant) {
+    case "neutral":
+      return isWheel
+        ? "var(--colorsSemanticNeutral500)"
+        : "var(--colorsSemanticNeutral200)";
+    case "gradient-grey":
+      return isWheel ? "#00D639" : "#0000001A";
+    case "gradient-white":
+      return isWheel ? "#00D639" : "var(--colorsActionMajorYang100)";
+    case "inverse":
+      return "var(--colorsActionMajorYang100)";
+    default:
+      return isWheel
+        ? "var(--colorsActionMajor500)"
+        : "var(--colorsActionMajor150)";
+  }
+};
+
+interface StyledSpinnerWrapperProps
+  extends Pick<LoaderSpinnerProps, "size" | "variant"> {
+  reduceMotion?: boolean;
+  size: LoaderSpinnerSizes;
+}
+
+export const StyledSpinnerWrapper = styled.div<
+  Pick<StyledSpinnerWrapperProps, "size">
```

Comment:
comment: it seems a bit odd to define an interface for this and then only pick part of it, I think `reduceMotion` isn't used anywhere anymore so it can just be removed and you can use the full interface etc

Reply:
Yeah good point, `reduceMotion` was originally added to change styling on the wrapper. As we're not applying any major styles to the wrapper and just the circle svgs we can remove it, and just pick straight from `LoaderspinnerProps` opposed to making a whole new interface